Desktop/utilities/qmlcacheconfiguration.cpp

```diff
@@ -0,0 +1,49 @@
+#ifdef linux
+
+#include "qmlcacheconfiguration.h"
+#include <QtGlobal>
+#include <QStandardPaths>
+#include "appinfo.h"
+#include "utilities/qutils.h"
+#include "log.h"
+
+
+void QMLCacheConfiguration::configureQMLCacheDir() {
+    //set cache environment variable
```

Comment:
small nitpick, we use 4-wide tabs as indentation in C++ and QML.
(And 2 spaces in R)